### DISCUSSION TITLE
Drivers - see the current version?

### DISCUSSION THREAD

**Discussion by alexbalandi** [seerge/g-helper#646] [General]
Recently switched to this after uninstalling armory crate (guess why, haha ;_;), looks very neat!
I see a lot of download options listed under "drivers" - is there a quick way to see current version so I understand whether I need to download them?

**Comment by seerge** [comment]
New ones are marked with bold res, if everything is green - its all good

**Reply by Kowken** [reply]
Hi ! Although most drivers are labeled green on my menu, some drivers are colored in grey: what does that mean ?
Thanks in advance for your answer.

**Reply by seerge** [reply]
I have changed that in 0.120. Grey is for software where you can't check current installed version.

I think it's better than just showing them as green (i.e. all ok) :)

**Reply by Kowken** [reply]
I see. I was afraid that I may have installation troubles but if I understand correctly, it's just that the app cannot access this information right ?
In that case, does it turn red when a new version is available for those ? Or does it stay grey since it can't check which version I have anyways ?

**Reply by seerge** [reply]
It will be always grey :) because for those apps (mostly) (and sometimes drivers) asus doesn't list device IDs they are related to, so it's not possible to check that anyhow. 

MyASUS also won't mark those as new/available for update

**Reply by Kowken** [reply]
Okay thanks for the clarification :)

**Comment by alexbalandi** [comment]
Thank you very much for clarification and quick answer ^_^
